udata_front/theme/gouvfr/datagouv-components/src/types/datasets.ts

```diff
@@ -25,7 +25,10 @@ export type BaseDataset = Owned & {
   tags: Array<string> | null;
   license: string;
   frequency: string;
-  temporal_coverage: string;
+  temporal_coverage: {
+    start: string;
+    end: string;
+  } | null;
   frequency_date: Date | null;
```

Comment:
I think this should be `string | null` BTW.

Reply:
You're right, the existing types were the ones from the publishing form but not the ones from the AP. We can change them to better fit their current use.